### ISSUE TITLE
The History tab in SilverStripe CMS does not display the Unpublish and Archive records

### ISSUE THREAD

**Issue by pack254** [tractorcow-farm/silverstripe-fluent#907]
### Module version(s) affected

7.1

### Description

This issue occurs in one of my projects and also affects SilverStripe CMS `5.2`. I tested it on a fresh installation of `silverstripe/installer` with `tractorcow/silverstripe-fluent` version `7.1`, and the problem persists. The "Unpublish" and "Archive" records are missing from the history tab, making it difficult to track these actions.

### How to reproduce

Steps to Reproduce:
1. Install `silverstripe/installer` on a local environment.
2. Add `tractorcow/silverstripe-fluent` version 7.1 to composer.json.
3. Create two locales.
4. Add a new page.
5. Fill in the required information for the new page, then save and publish it.
6. Then unpublish or archive the new page.
7. Check the "History" tab. You will notice that it does not show records for "Unpublish" or "Archive" actions.

### Additional Context

![Screenshot 2024-11-19 at 10 41 23 AM](https://github.com/user-attachments/assets/b7137cfc-df0e-482b-92fd-ff8ef618c3a9)

### Validations

- [X] Check that there isn't already an issue that reports the same bug
- [X] Double check that your reproduction steps work in a fresh installation of [`silverstripe/installer`](https://github.com/silverstripe/silverstripe-installer) (with any code examples you've provided)

**Comment by GuySartorelli** [comment]
Just to be clear: I assume you also tried this in an installation _without_ the fluent module and weren't able to reproduce it?

**Comment by pack254** [comment]
> Just to be clear: I assume you also tried this in an installation _without_ the fluent module and weren't able to reproduce it?

Yep, your assumption is right @GuySartorelli. without the Fluent module, these steps cannot be reproduced. 

It will be shown all the records correctly.
![Screenshot 2024-11-19 at 11 35 41 AM](https://github.com/user-attachments/assets/fe0c976a-b76e-45a8-81fb-041c28ecee6d)
